### DISCUSSION TITLE
Modifying closing hint

### DISCUSSION THREAD

**Discussion by simonmandlik** [andymass/vim-matchup#315] [Q&A]
Can I somehow disable / modify the ending `◀ if` hint here?

<img width="181" alt="image" src="https://github.com/andymass/vim-matchup/assets/16707112/b4100a6f-ff9c-49d9-a084-fd3cc4d3a4e5">

**Comment by andymass** [comment]
Discussed here, https://github.com/andymass/vim-matchup/issues/316

https://github.com/andymass/vim-matchup/blob/582cbc8c2dc311deb7a03f7d505d52544c36d4b6/doc/matchup.txt#L295
